devenv/datasources.yaml

```diff
@@ -71,6 +71,7 @@ datasources:
 
   - name: gdev-testdata
     isDefault: true
+    uid: gdev-testdata
```

Comment:
@yuri-tceretian while a nice chance , we need to update all gdev dashboards as well, but even then this would break hundreds of developer dashboards that all refer to this data source as "PD8C576611E62080A"